apps/provider-stub/.env.example

```
@@ -0,0 +1,22 @@
+NODE_ENV=development
+PORT=4001
+
+# --- MongoDB (adjust if needed) ---
+MONGO_URL_DEV=mongodb://localhost:27017/hcx_provider
+
+# --- NHCX gateway ---
+NHCX_BASE_URL=https://apisbx.abdm.gov.in/hcx/v1
+
+# --- ABDM Session (Provider) ---
+SESSION_API_URL=https://dev.abdm.gov.in/api/hiecm/gateway/v3/sessions
+ABDM_CLIENT_ID=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The ABDM_CLIENT_ID key should go before the SESSION_API_URL key

(UnorderedKey)